pkg/sources/gitlab/gitlab.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const SourceType = sourcespb.SourceType_SOURCE_TYPE_GITLAB
+const cloudBaseURL = "https://gitlab.com/"
```

Comment:
Question: would `gitlabHostedBaseURL` be a better var name for this? `cloudBaseURL` is a bit confusing to me as `cloud` could mean a gitlab user's self-hosted gitlab instance

Reply:
amusingly, i had the opposite reaction! but i see your point. i wonder if it's actually clearer to _not_ use a const and just inline the literal where we need it? (or maybe make the const local to the function? can you do that?)

either way, a comment is probably warranted if you can't find a magically perfectly clear identifier.